plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/FileManagementUtil.java

```diff
@@ -144,43 +105,15 @@ static private void addFolderToZip(String path, String srcFolder, ZipOutputStrea
         File folder = new File(srcFolder);
         String fileListArray[] = folder.list();
         int i = 0;
-        while (i < fileListArray.length) {
-            String newPath = folder.getName();
-            if (!path.equalsIgnoreCase("")) {
-                newPath = path + File.separator + newPath;
-            }
-            addToZip(newPath, srcFolder + File.separator + fileListArray[i], zip);
-            i++;
-        }
-
-    }
-
-    public static void deleteDirectories(File dir) {
-        File[] children = dir.listFiles();
-        if (children != null) {
-            for (File aChildren : children) {
-                if (aChildren.list() != null && aChildren.list().length > 0) {
-                    deleteDirectories(aChildren);
-                } else {
-                    aChildren.delete();
+        if (fileListArray != null) {
+            while (i < fileListArray.length) {
+                String newPath = folder.getName();
+                if (!path.equalsIgnoreCase("")) {
+                    newPath = path + File.separator + newPath;
```

Comment:
Thanks. Fixed in https://github.com/wso2/devstudio-tooling-bps/pull/30/commits/54abcc8d24b081f392bda8ae8eeff1023aae34a9